internal/provider/template_test.go

```diff
@@ -93,7 +93,7 @@ provider "scaffolding" {
 		},
 	}
 
-	result, err := tpl.Render("testdata/test-provider-dir", "testTemplate", "test-provider", "test-provider", "Resource", "provider.tf", "provider.tf", &schema)
+	result, err := tpl.Render("testdata/test-provider-dir", "testTemplate", "test-provider", "test-provider", "Resource", "provider.tf", "", "", &schema)
```

Comment:
Internally, the render function ignores files that don't exist, which the original `provider.tf` string was triggering. So I made it explicitly blank to avoid confusion 😆